### DISCUSSION TITLE
Setting the temperature via jarvis seems not to work.

### DISCUSSION THREAD

**Discussion by heiko17** [Zefau/ioBroker.jarvis#2067] [Q&A] [answered]
If I try to set the Thermostat setpoint_heating via input field it does not work.
A error message appears in log. 
`Thermostat Setpoint: "setpoint" must be of type "number", received "string" (ZW0322)`
Not sure if this is a problem with jarvis, zwave2 or with my configuration. 
jarvis version: 3.0.13
zwave2 version: 3.0.3

Setting the hermostat setpoint from iobroker object view work without problems.
Also setting the setpoint via jarvis "LevelBody" works.

**Comment by Zefau** [comment, marked as answer]
This is fixed in v3.1, which will be released in beta channel soon. Currently available on GitHub (installation see https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/v3-fuer-einsteiger/installation-jarvis-v3)
